python/paddle/fluid/framework.py

```diff
@@ -34,6 +34,8 @@
 from . import core
 from . import unique_name
 
+import paddle
```

Comment:
DO NOT import paddle here, this may cause circle deps

Reply:
done